changelog/1702.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
```suggestion
```release-note:change
```

Let's also make this a change so people are aware of it. I don't think feature is quite correct, but it might break people.

Reply:
We don't actually have `feature` vs `improvement` labels; that one was assigned by the default issue template fwiw. :-)